kong/db/schema/others/declarative_config.lua

```diff
@@ -409,10 +409,24 @@ local function get_key_for_uuid_gen(entity, item, schema, parent_fk, child_key)
     return
   end
 
-  if schema.endpoint_key and
-     item[schema.endpoint_key] ~= nil then
+  if schema.endpoint_key and item[schema.endpoint_key] ~= nil then
+    local key = item[schema.endpoint_key]
+
+    -- If this item has foreign keys with on_delete "cascade", it is infered
+    -- that its endpoint is not necesserally unique, so its key must be composed
+    -- by parent's key, avoiding that it is overwritten by identical endpoints
+    -- under other parents.
```

Comment:
Typos: "infered" (inferred) and "necesserally" (necessarily)

Reply:
Fixed that, thanks!